internal/service/federatedsettingsorgconfig/data_source_federated_settings_connected_org.go

```diff
@@ -3,7 +3,6 @@ package federatedsettingsorgconfig
 import (
```

Comment:
nit: do we know why we name files related to `mongodbatlas_federated_settings_org_config` as `federated_settings_connected_org`? Was getting confused 😄

Reply:
aahhh. I had to read this a few times to understand 😆 No idea why we have those file names.
Is it worth changing them?

Reply:
can be a nice follow up, no problem in having in separate PR to avoid more changes in this one.